test/go/ferry_test.go

```diff
@@ -18,11 +18,6 @@ func (t *FerryTestSuite) SetupTest() {
 	t.GhostferryUnitTestSuite.SetupTest()
 }
 
-func (t *FerryTestSuite) TearDownTest() {
```

Comment:
Why was this removed?

Reply:
My mistake, I resolved a previous clarification pre-emptively. See [here](https://github.com/Shopify/ghostferry/pull/253#discussion_r580348436) for more context.

TL;DR This teardown was specific to the _single_ test that lived in this suite, so it went uncaught. The teardown doesn't apply to all tests, so I removed it.